projects/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from projects.models import Project
+
+
+class ProjectAdmin(admin.ModelAdmin):
```

Comment:
its that class is needed when we don't have any implementation?

Reply:
done